### ISSUE TITLE
Foreign Key problem

### ISSUE THREAD

**Issue by iChicory** [ibmdb/ruby-ibmdb#35]
I'm using ruby 2.1.5, rails 4.2.1, ibmdb 2.6.0
Have tables :users and :requests
When I make migration:
```
class AddForeignKeys < ActiveRecord::Migration
  def change
    add_foreign_key(:requests, :users, on_delete: :cascade)
  end
end
```
I have next result:
```
spring rake --trace db:migrate
** Invoke db:migrate (first_time)
== 20150701155334 AddForeignKeys: migrating ===================================
** Invoke environment (first_time)
-- add_foreign_key(:requests, :users, {:on_delete=>:cascade})
** Execute environment
** Invoke db:load_config (first_time)
** Execute db:load_config
   -> 0.0001s
== 20150701155334 AddForeignKeys: migrated (0.0004s) ==========================

** Execute db:migrate
** Invoke db:_dump (first_time)
** Execute db:_dump
** Invoke db:schema:dump (first_time)
** Invoke environment 
** Invoke db:load_config 
** Execute db:schema:dump

Process finished with exit code 0
```

BUT, when I check db2 tables via dbvisualizer - there is no any records about foreign keys

**Comment by arvindgu** [comment]
Currently ibm_db gem does not support foreign keys, this is why foreign keys do not get created in database. We have added this in our todo list.

**Comment by arvindgu** [comment]
The foreign keys are now supported in the latest released 2.6.1.

**Comment by iChicory** [comment]
Thank you
